cfg/lynx.cfg

```diff
@@ -42,4 +43,4 @@ FEATURES {
             count   = __INTERRUPTOR_COUNT__,
             segment = RODATA,
             import  = __CALLIRQ__;
-}
+}
```

Comment:
You seem to have removed the final linefeed here. It' scommon practise to end all source files with a linefeed.